src/app/profile/offline-meditation/offline-meditation.html

```diff
@@ -34,7 +34,10 @@
           [mdDatepicker]="picker"
           [max]="today"
           placeholder="Date"
-          [disabled]="sending">
+          (focus)="picker.open()"
+          [disabled]="sending"
+          type="text"
+          readonly="true">
```

Comment:
It's only meant for displaying the date of the picker. The model is a Date object and changes to String if the user is allowed to input something. I could add that you can also enter the date by hand, but that would require additional format checking and date conversion. I assumed that everybody is able to use the Date Picker for convenience reasons.